testing/src/main/java/io/grpc/testing/GrpcCleanupRule.java

```diff
@@ -75,6 +76,7 @@ public final class GrpcCleanupRule extends ExternalResource {
 
   private final List<Resource> resources = new ArrayList<>();
   private long timeoutNanos = TimeUnit.SECONDS.toNanos(10L);
+  @SuppressWarnings("SystemTickerUnreliable")
```

Comment:
This is internal-only. I don't think it makes sense to have here, because you can't really maintain it and notice the results. (Even in google3 it sorta doesn't make sense, because it doesn't show up in a build.)